docs/src/lib/transform.ts

```diff
@@ -12,19 +12,24 @@ export function transformProject(project: typeof DATA) {
   const { children } = project;
   invariant(children !== undefined, "The typedoc output is empty!");
 
-  const functions = children
-    .filter(({ kind }) => kind === ReflectionKind.Function)
-    .map(transformFunction);
+  const functions = children.filter(
+    ({ kind }) => kind === ReflectionKind.Function,
+  );
+
+  const functionNames = new Set(functions.map(({ name }) => name));
 
-  return addCategories(project, functions.filter(isDefined.strict));
+  return addCategories(
+    project,
+    functions
+      .map((function_) => transformFunction(function_, functionNames))
```

Comment:
I prefer an abbreviation over adding the suffix underscore.
```suggestion
      .map((func) => transformFunction(func, functionNames))
```